ci/configure-aws.sh

```diff
@@ -48,22 +48,22 @@ function assume_iam_role() {
 
     export AWS_ACCESS_KEY_ID=$(echo ${CREDS_JSON}     | jq ".Credentials.AccessKeyId" --raw-output)
     export AWS_SECRET_ACCESS_KEY=$(echo ${CREDS_JSON} | jq ".Credentials.SecretAccessKey" --raw-output)
-    export AWS_SESSION_TOKEN=$(echo ${CREDS_JSON}    | jq ".Credentials.SessionToken" --raw-output)
+    export AWS_SESSION_TOKEN=$(echo ${CREDS_JSON}     | jq ".Credentials.SessionToken" --raw-output)
 }
 
 # Clear the environment variables set after calling assume_iam_role to get back to
 # the initial state. (Using the default credentials or ${AWS_PROFILE}.)
 function unassume_iam_role() {
-    unset {AWS_ACCESS_KEY_ID,AWS_SECRET_ACCESS_KEY,AWS_SECURITY_TOKEN}
+    unset {AWS_ACCESS_KEY_ID,AWS_SECRET_ACCESS_KEY,AWS_SESSION_TOKEN}
 }
 
 # Check information.
 if [ -z ${AWS_ACCESS_KEY_ID} ]; then
     echo "ERROR: AWS_ACCESS_KEY_ID not set. Something is not right. (AWS creds not set?)"
 fi
 
-if [ ! -z ${AWS_SECURITY_TOKEN} ]; then
-    echo "ERROR: AWS_SECURITY_TOKEN is set. Something is not right. (In an assumed role?)"
+if [ -n "${AWS_SECURITY_TOKEN}" ] || [ -n "${AWS_SESSION_TOKEN}" ]; then
+    echo "ERROR: AWS_SECURITY_TOKEN or AWS_SESSION_TOKEN is set. Something is not right. (In an assumed role?)"
```

Comment:
Should we add an `exit 1` or something here since the message is printed as an error? Not sure if that was not done intentionally previously.